core/store/migrate/migrations/0107_solana_chains_nodes.sql

```diff
@@ -0,0 +1,26 @@
+-- +goose Up
```

Comment:
hm looks like we are forcing all node ops to delete solana jobs then re-add them? Seems like we could extract what we need from the v1 relay config and create the appropriate chain/nodes needed for a smooth upgrade

Reply:
even if we didn't support a smooth upgrade, I think we want to clean up the now invalid fields in the relay json https://github.com/smartcontractkit/chainlink-solana/pull/250/files#diff-7ce0d6e991fe4a942d82f97d6679dce69277744382b752410246c9dbc5f2ae1dL165 etc.

Reply:
yeah we are, node ops aren't running anything at the moment so it didn't make sense to add migration code